app/renderers/oembed_attribute_renderer.rb

```diff
@@ -1,8 +1,12 @@
+# frozen_string_literal:true
+
 class OembedAttributeRenderer < Hyrax::Renderers::AttributeRenderer
-    def attribute_value_to_html(value)
-      html = ""
-      resource = OEmbed::Providers.get(value)
-      html << resource.html
-      return html
-    end
+  # TODO: Offload the loading of the OEmbed value to the client,
+  # render only the link to client-side.
+  def attribute_value_to_html(value)
+    resource = OEmbed::Providers.get(value)
+    resource.html
+  rescue OEmbed::NotFound
```

Comment:
Won't matter after #113 , but this breaks bubbling the error up out of the renderer into a context that can access the work controller/id to generate the OmbedError object.
Error is later rescued in `app/views/hyrax/base/_attribute_rows.html.erb` which is awful, but like I said, fixed in PR #192

Reply:
Non-issue. Fixed later.